service/src/main/java/io/camunda/service/license/CamundaLicense.java

```diff
@@ -16,11 +16,11 @@
 
 public class CamundaLicense {
 
+  public static final String CAMUNDA_LICENSE_ENV_VAR_KEY = "CAMUNDA_LICENSE_KEY";
   private static final Logger LOGGER = LoggerFactory.getLogger(CamundaLicense.class);
   private static final String UNKNOWN_LICENSE_TYPE = "unknown";
```

Comment:
if you proceed with enums, I don't think this would be necessary (also happy with a non-enum solution)

Reply:
I like the idea of enums now that I have it implemented, its guaranteeing that valid types are going to be used, and all else gets bucketed into `unknown`

Since this string is used in multiple places, I'm inclined to keep it here